.coveragerc

```diff
@@ -1,3 +1,4 @@
 [run]
 parallel = True
 data_file = /tmp/${USER}/cov/coverage.db
+omit = */site-packages/*
```

Comment:
Can you move this to a separate PR entirely?

Reply:
#2102